### PR TITLE
Add option to use bold text in terminals

### LINKED ISSUE #182: Allow bold text for prompts etc
ConsoleZ (right hand side) has an option to enable bold text for prompts etc (which is how the native `gnome-terminal` looks). Is this possible (or could it be implemented :smile:)?

screenshot_win10_2019-01-20_16 17 50

### PR DESCRIPTION
**Pull request by ericcornelissen**
Closes #182

Adds a toggle to the Terminal Page to enable/disable bold text in terminals. Currently, from my testing, when `BoldText` is enabled there is no real distinction between normal and bold text in most fonts, I also addressed this in [this comment](https://github.com/felixse/FluentTerminal/issues/182#issuecomment-455901102).

### Alternative implementation
Currently the conversion from boolean to `fontWeight` and `fontWeightBold` value is performed 2x each, alternatively we could use transformer functions, e.g. ⬇️, but this seems like overkill to me.

```javascript
term.setOption('fontWeight', boldToFontWeight(options.boldText));
term.setOption('fontWeightBold', boldToFontWeightBold(options.boldText));

...

function boldToFontWeight(boldText) {
  return options.boldText ? 'bold' : 'normal';
}
function boldToFontWeightBold(boldText) {
  return options.boldText ? 'bolder' : 'bold';
}
```

### CODE REVIEW

File: FluentTerminal.App.ViewModels/Settings/TerminalPageViewModel.cs [hunk 1]
```diff
@@ -136,6 +136,20 @@ public string ScrollBackLimit
             }
         }
 
+        public bool BoldText
```
Comment (felixse) [inline FluentTerminal.App.ViewModels/Settings/TerminalPageViewModel.cs]: Please also add to TerminalPageViewModel.RestoreDefaults()

File: FluentTerminal.Models/TerminalOptions.cs [hunk 2]
```diff
@@ -8,6 +8,8 @@ public class TerminalOptions
 
         public int FontSize { get; set; }
 
+        public bool BoldText { get; set; }
```
Comment (felixse) [inline FluentTerminal.Models/TerminalOptions.cs]: Can you please specify a default value for this in DefaultValueProvider?

**Comment by ericcornelissen** [comment]
The build failure seems to be caused by [the certificate being invalid](https://ci.appveyor.com/project/felixse/fluentterminal/builds/21909961#L826), the same is true for the [latest build on master](https://ci.appveyor.com/project/felixse/fluentterminal/builds/21906620#L825) even though the certificate was updated in that commit for #185 🤔

**Comment by felixse** [comment]
That's my fault. The CI is using some encrypted certificates I should also have updated, but my plan is to migrate the CI to Azure Pipelines in the next days.

**Pull request merged**